client/src/pages/ViewSubmissions.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Image } from './Image';
+
+export class ViewSubmissions extends React.Component {
+	constructor(props) {
+		super(props)
+		this.state = {
+			submissions: [], 
+			submissionKeys: []
+		}
+   	this.viewAllSubmissions = this.viewAllSubmissions.bind(this)
+	}
+
+	componentDidMount() {
+    this.viewAllSubmissions()
```

Comment:
do you use a linter? discussing proper lint settings with teammates may benefit code readability :)

Reply:
I don't, but I can see now why I should haha! I'll bring that up with the team :)